uni/lib/controller/fetchers/schedule_fetcher/schedule_fetcher.dart

```diff
@@ -28,9 +28,8 @@ abstract class ScheduleFetcher extends SessionDependantFetcher {
 
 /// Stores the start and end dates of the week and the current lective year.
 class Dates {
```

Comment:
Not related to this PR, but this has got to be the most creative class I've ever seen :)